assets/styles/base/_mixins.scss

```diff
@@ -146,7 +146,8 @@
 
     &.error {
       border: solid 1px var(--error);
-      input, .selected {
+
```

Comment:
This removes all indication of an error when the field is focused. If a user can clear the error whilst focused I think we should retain some kind of red

No Hover or Focus
![image](https://user-images.githubusercontent.com/18697775/159898769-7c95b2a6-2c04-49a9-8d84-7fe7bc6d970d.png)

Hover no Focus
![image](https://user-images.githubusercontent.com/18697775/159898838-fd75733c-9575-4499-a1f5-f1137291000d.png)

Focus
![image](https://user-images.githubusercontent.com/18697775/159898921-a6d0a6a1-d98c-4c1f-9ff3-1c34a63f5572.png)